pubspec.yaml

```diff
@@ -61,7 +61,7 @@ dependencies:
   sqlite3_flutter_libs: ^0.5.13
   url_launcher: ^6.1.11
   url_launcher_android: ">=6.1.0"
-  video_player: 2.9.5 # TODO unpin and upgrade to latest version
+  video_player: ^2.10.0
```

Comment:
What does our Git history say about the reason for the pin; was it just the "small changes to the public API" mentioned and addressed here, or was there another reason? That would be helpful to make clear in the commit message.

Reply:
Updated the commit message.